src/platform/file_manager_cache.h

```diff
@@ -0,0 +1,28 @@
+#ifndef FILE_MANAGER_CACHE_H
+#define FILE_MANAGER_CACHE_H
+
+#if defined(__vita__) || defined(__SWITCH__)
```

Comment:
Suggestion: create define for USE_FILE_CACHE:
```suggestion
#if defined(__vita__) || defined(__SWITCH__)
#define USE_FILE_CACHE
#endif
```
In the other places you can then just use `#ifdef USE_FILE_CACHE` instead of repeating the vita/switch define checks. This makes it easier to add caching to other platforms if that's ever necessary.